code/API_definitions/Test_definitions/Traffic_Influence_Test.feature

```diff
@@ -0,0 +1,78 @@
+#/*- ---license-start
+#* CAMARA Project
+#* ---
+#* Copyright (C) 2022 - 2023 Contributors | Deutsche Telekom AG to CAMARA a Series of LF Projects, LLC
+#* The contributor of this file confirms his sign-off for the
+#* Developer Certificate of Origin (http://developercertificate.org).
+#* ---
+#* Licensed under the Apache License, Version 2.0 (the "License");
+#* you may not use this file except in compliance with the License.
+#* You may obtain a copy of the License at
+#*
+#*      http://www.apache.org/licenses/LICENSE-2.0
+#*
+#* Unless required by applicable law or agreed to in writing, software
+#* distributed under the License is distributed on an "AS IS" BASIS,
+#* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#* See the License for the specific language governing permissions and
+#* limitations under the License.
+#* ---license-end
+#*/
+
+@TI @TISanity
+Feature: Automated Traffic Influence API Test
+
+  @TI_Resource_LCM_Mandatory_Parameters_Valid
+  Scenario: Manage Traffic Influence (TI) Resource with mandatory parameters
```

Comment:
Are there any preconditions that need to be declared (with the 'Given' keyword)?

Reply:
I added as Given:
- the usage of EdgeClouAPI to get applicationId
- the retrieval of Regions and Zones